.eslintrc.json

```diff
@@ -65,7 +65,16 @@
     "prefer-exponentiation-operator": "error",
     "prefer-rest-params": "error",
     "prefer-spread": "error",
-    "space-in-parens": ["error", "never"]
+    "space-in-parens": ["error", "never"],
```

Comment:
nit: let's keep the alphabetic order for rules (we will remove them later when we adopt new config).

Reply:
Done.